src/v2/Apps/Fair/Components/FairHeader/FairHeader.tsx

```diff
@@ -10,18 +10,53 @@ import {
   Spacer,
 } from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
 import { FairHeader_fair } from "v2/__generated__/FairHeader_fair.graphql"
 import { FairTimingFragmentContainer as FairTiming } from "./FairTiming"
-import { FairHeaderImageFragmentContainer as FairHeaderImage } from "./FairHeaderImage"
 import { FairHeaderIconFragmentContainer } from "./FairHeaderIcon"
+import {
+  FairEditorialFragmentContainer,
+  FAIR_EDITORIAL_AMOUNT,
+} from "../FairEditorial"
+import { FairCollectionsFragmentContainer } from "../FairCollections"
+import { FairFollowedArtistsFragmentContainer } from "../FairFollowedArtists"
+import { useSystemContext } from "v2/System"
 import { RouterLink } from "v2/System/Router/RouterLink"
 
+const TextWithNewlines = styled(Text)`
+  white-space: pre-wrap;
+`
+
+interface FairInfoSectionProps {
+  label?: string
+  info: string
+  isHTML?: boolean
+}
+
+const FairInfoSection: React.FC<FairInfoSectionProps> = ({
+  label,
+  info,
+  isHTML = false,
+}) => (
+  <>
+    {label && <Text variant="md">{label}</Text>}
+    {isHTML ? (
+      <HTML variant="md" html={info} />
+    ) : (
+      <TextWithNewlines variant="md">{info}</TextWithNewlines>
```

Comment:
What's an example of when this is desired?

Reply:
@dzucconi  For example, `<TextWithNewLines />` used for display location which is a string with `\r\n` symbols. So if we use `<TextWithNewLines />` it will be rendered the string below:
![image](https://user-images.githubusercontent.com/56556580/126759234-48db5f57-8c4d-4ea9-8277-131256fc92c0.png)

In other way we receive one line string:

![image](https://user-images.githubusercontent.com/56556580/126759196-944a3510-6311-4aa5-a6da-46640a0bd6a5.png)